src/sentry/integrations/github/repository.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import absolute_import
+
+import logging
+import six
+
+from sentry.models import Integration, Organization
+from sentry.plugins import providers
+
+WEBHOOK_EVENTS = ['push', 'pull_request']
+
+
+class GitHubRepositoryProvider(providers.IntegrationRepositoryProvider):
+    name = 'GitHub Apps'
+    auth_provider = 'github'
+    logger = logging.getLogger('sentry.plugins.github')
+
+    def get_config(self, organization):
+        choices = []
+        for i in Integration.objects.filter(organizations=organization, provider='github'):
+            choices.append((i.id, i.name))
+
+        return [
+            {
+                'name': 'installation',
+                'label': 'Github Installation',
+                'type': 'choice',
+                'choices': choices,
+                'initial': choices[0][0],
```

Comment:
i also should have noticed this before, but you should add a check here `choices and choices[0][0]` to avoid an index error (otherwise i think the repo page will be unusable for us if we don't have a github apps integration set up for our org)